### PR TITLE
{bio} [foss/2018a] BWISE (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
Needs easybuilders/easybuild-easyblocks#1497

### CODE REVIEW

File: easybuild/easyconfigs/b/BWISE/BWISE-20180820-foss-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    '072deda58732c452bd8e82e4b4efd0b9763679726a94c589daf483942f460da6',  # bgreat2-20180425.tar.gz
+    '21e1288e246171ab76f4fd1d9c4a7bab24ac5850087b313403c888b061326a60',  # btrim-20180503.tar.gz
+]
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('BCALM', '2.2.0'),
+]
+
+sanity_check_paths = {
```
Comment (boegel) [inline easybuild/easyconfigs/b/BWISE/BWISE-20180820-foss-2018a-Python-3.6.4.eb]: @wpoely86 This should be handled by the easyblock?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic1 - Linux centos linux 7.5.1804, AMD Opteron(tm) Processor 6134, Python 2.7.5
See https://gist.github.com/3f2a3e0cbad7f7689b5b3a70f965ef96 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic1 - Linux centos linux 7.5.1804, AMD Opteron(tm) Processor 6134, Python 2.7.5
See https://gist.github.com/a6fa3bfd71cb443bfd7e7c547462daeb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2596.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c2e8ca384931d0e0c3d2685d3c27f4d5 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**